apps/desktop/src/mods/ModBox.tsx

```diff
@@ -138,6 +141,21 @@ export function ModBox(props: IModBoxProps) {
     });
   };
 
+  const updated = (updated: string) => {
+    var diff =
+      (new Date().valueOf() - new Date(updated).valueOf()) / (1000 * 3600 * 24);
+    return diff < 1 ? 'today' : Math.ceil(diff) + ' days ago';
+  };
+
+  const isUpdatedSinceGameUpdate = (updated: string) => {
+    if (
+      Math.ceil(new Date(updated).valueOf() / 1000) > Number(LatestGameUpdate)
+    ) {
+      return true;
+    }
+    return false;
```

Comment:
always with dayjs this could be simplified to `dayjs(updated).isAfter(dayjs(latestGameUpdate))`